README.md

```diff
@@ -11,19 +11,81 @@ $ go get github.com/daixiang0/gci
 ```
 
 ## Usage
+GCI supports three modes of operation
+```shell
+$ gci print -h
```

Comment:
Could we merge `print` and `write` together as `write` seems more like an option for `print`, how about `gci format`?

Reply:
The reason for choosing `write` and `print` is that I think they make it immediately clear to the user what they do. With `format` I would not be sure if it reformats the file or just prints a formatted version.
However I could easily add an alias like there is with `overwrite` for `write`.

Reply:
In fact `print` may mislead as well, let's make it clear and easy for users:
- `gci <files>` to print formatted version
- `gci -w <files>` to write formatted version

Keep same logic with `goimports` makes users easily migrate, what do you think?

Reply:
The "old" style were you can simply do `gci <files>` and `gci -w <files>` is still fully present and functional to provide backwards compatibility. Thus if some users prefer that format they can use it.
However I did not want to modify this style with breaking changes by introducing the options for sections there.
I also did not add the help output for this to the Readme in order not to confuse new users who can start with the new subcommands. But if you just run `gci -h` you still see all the old options:
```
Usage:
  gci [-diff | -write] [-local localPackageURLs] path... [flags]
  gci [command]

Available Commands:
  completion  Generate the autocompletion script for the specified shell        
  diff        Prints a git style diff to STDOUT
  help        Help about any command
  print       Outputs the formatted file to STDOUT
  write       Formats the specified files in-place

Flags:
  -d, --diff            display diffs instead of rewriting files
  -h, --help            help for gci
  -l, --local strings   put imports beginning with this string after 3rd-party p
ackages, separate imports by comma
  -v, --version         version for gci
  -w, --write           write result to (source) file instead of stdout
```
I think this is the best compromise for adding this new functionality. This way the cli that previous users are accustomed to is still present. If you want to use the new functionality you can just use the new commands.
Another benefit of the new subcommands is that they have autocomplete support. Just writing `gci w<TAB>` which gets autocompleted is even faster than writing `gci -w`.

Reply:
Great, let's make it land.